security/security-spring/src/main/java/org/cbioportal/security/spring/authentication/social/CustomUserDetailsService.java

```diff
@@ -1,4 +1,4 @@
-package org.cbioportal.security.spring.authentication.googleplus;
+package org.cbioportal.security.spring.authentication.social;
```

Comment:
Note to @n1zea144 @sheridancbio @ao508 this package change will require us to update our private security context file.